ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/BlockProcessorUtil.java

```diff
@@ -427,13 +427,15 @@ static boolean verify_randao(
    */
   private static boolean verify_bitfields_and_aggregate_signature(
```

Comment:
@jrhea I wouldn't mind a double check of my logic here. See the first 2 lines in the codeblock here: https://github.com/ethereum/eth2.0-specs/blob/0.4.0/specs/core/0_beacon-chain.md#attestations-1

Reply:
ya looks right dude